components/browser/state/src/main/java/mozilla/components/browser/state/state/content/PermissionHighlightsState.kt

```diff
@@ -14,6 +15,7 @@ import kotlinx.android.parcel.Parcelize
  * @property isAutoPlayBlocking indicates if the autoplay setting
  * disabled some web content from playing.
  */
+@SuppressLint("ParcelCreator") // Suppress false positive
 @Parcelize
 data class PermissionHighlightsState(
```

Comment:
@Amejia481 Does `PermissionHighlightsState` actually need to be `Parcelable`?

Reply:
We are using it to pass data from on [Fragment to another in Fenix](https://github.com/mozilla-mobile/fenix/blob/54db8f3fde08532b05a7f62f0b648a23aa693d0b/app/src/main/res/navigation/nav_graph.xml#L779), but it if needed we could could remove it can just pass the single value.